.registryrc

```diff
@@ -1 +1 @@
-29f4901c1d1a63944b0f5f2ee50f499568260004
+5e1d983a8ac3cafc3bad918bda9b609e513034b2
```

Comment:
This commit hash is not in `main` of registry.

Reply:
yup - will be updating after https://github.com/hyperlane-xyz/hyperlane-registry/pull/138 gets merged